Header header
string node
string[] topicList
float32[] topicLatency
string[] monitorList
string[] attachedTopics
float32[] monitorUsage
int32[] monitorATCF
uint32[] eventsInBuffer
float32 wholeNodeUsage
float32 wholeMachineUsage
float32 availableUsage
float32 thisMachineMaxUsage
int64 machineCPUfreq
float32 milpUsageThreshold
